src/platform/web/ui/session/room/TimelineView.ts

```diff
@@ -57,16 +57,27 @@ function findFirstNodeIndexAtOrBelow(tiles: HTMLElement, top: number, startIndex
     return 0;
 }
 
+interface TimelineViewOpts {
+    viewClassForTile: ViewClassForEntryFn
+    stickToBottom: boolean
+}
 export class TimelineView extends TemplateView<TimelineViewModel> {
 
     private anchoredNode?: HTMLElement;
     private anchoredBottom: number = 0;
     private stickToBottom: boolean = true;
     private tilesView?: TilesListView;
     private resizeObserver?: ResizeObserver;
+    private viewClassForTile: ViewClassForEntryFn;
 
-    constructor(vm: TimelineViewModel, private readonly viewClassForTile: ViewClassForEntryFn) {
+    constructor(vm: TimelineViewModel, {
+        viewClassForTile,
+        stickToBottom = true
+    }: TimelineViewOpts) {
```

Comment:
I've made this an options object because I forsee more options here (see future notes in PR description)